src/bip/plugins/juliet/parser.py

```diff
@@ -224,28 +223,30 @@ def read_packet(self, buf: RawIOBase):
         
 
     def process_packet(self, packet_type: int, indicators: int, payload: bytes):
+        # data is framed 1 frame per vita 49.2 packet
+         # so frame_index and packet_index are the same
         if packet_type == 0b0001:
```

Comment:
Just me but this is a bit of a magic number it would be good to set it up as a variable with some comments so that it is understandable.